lib/zlib.toit

```diff
@@ -704,29 +704,29 @@ class InflaterBackend implements Backend_:
         if extra-repeats != 0:
           last := lengths_[counter_ - 1].bit-len
           extra-repeats.repeat: add-symbol.call last
-        length-code := next_ meta-table_
-        if length-code < 0: return NEED-MORE-DATA_
-        if length-code < 16:
-          add-symbol.call length-code
-        else:
-          last := 0
-          repeats := 3
-          if length-code == 16:
-            last = lengths_[counter_ - 1].bit-len
-            pending-bits_ = 2
-          else if length-code == 17:
-            pending-bits_ = 3
+        if counter_ != lengths_.size:
```

Comment:
Should this be an `else:` on the check in line 726?

Reply:
No, add-symbol can update counter_.